hugo/content/contact.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Contact
+toc: true
+---
+
+# Contact
+
+You can contact us in several ways:
+
+## Community Channels
+
+* Our [issue tracker on GitHub](https://github.com/mumble-voip/mumble/issues)
+* [Discussions on GitHub](https://github.com/mumble-voip/mumble/discussions)
+* IRC: [#mumble on irc.freenode.net](irc://irc.freenode.org/mumble)
+* Chat: [Mumble room on Riot/Matrix](https://riot.im/app/#/room/#mumble:matrix.org) (shares messages with IRC)
+* Our [Forums](https://forums.mumble.info/)
+
+## Email
+
+**General contact:**
+
+contact [at] mumble [dot] info
+
+
+**For security-relevant topics:**
+
+security [at] mumble [dot] info
+
+
+**For legal matters:**
+
+<!-- add special adress for legal matters? -->
```

Comment:
Do you already have a special emailadress for this and if not, do you want to create one?

Reply:
I don't think so. To my knowledge we never needed that and if there really is something to discuss they can simply use the contact mail address...

Reply:
Ok.